lib/livebook/session.ex

```diff
@@ -3171,6 +3171,10 @@ defmodule Livebook.Session do
 
   # Traverse composite outputs
 
+  defp normalize_runtime_output(%{type: :plain_text} = plain_text) do
```

Comment:
Nitpick: the comment above says "Traverse composite outputs" which does not apply here. So let's move it after the composite outputs or remove the comment :D